l16/dev/l16_runner.sh

```diff
@@ -0,0 +1,196 @@
+}
+
+function start_validator {
+	echo "starting validator node..."
+
+	nohup ./bin/validator \
+	--accept-terms-of-use \
+	--rpc \
+	--wallet-password-file $BASE_PATH/kintsugi/datadir/validator/password.txt \
+	--chain-config-file $BASE_PATH/kintsugi/config/config.yaml \
+	--wallet-dir $BASE_PATH/kintsugi/datadir/validator/prysm \
+	--verbosity debug \
+	--datadir $BASE_PATH/kintsugi/datadir/validator>$$BASE_PATH/kintsugi/datadir/validator/logs/validator_"$RUN_DATE".log 2>&1 & 
+
+}
+
+function start_deposit_validator {
+	echo "starting deposit validator node..."
+
+	nohup ./bin/validator \
+	--accept-terms-of-use \
+	--rpc \
+	--rpc-port 7002 \
+	--monitoring-port 8082 \
+	--grpc-gateway-port 7502 \
+	--wallet-password-file $BASE_PATH/kintsugi/datadir/validator-deposit/password.txt \
+	--chain-config-file $BASE_PATH/kintsugi/config/config.yaml \
+	--wallet-dir $BASE_PATH/kintsugi/datadir/validator-deposit/prysm \
+	--verbosity debug \
+	--datadir $BASE_PATH/kintsugi/datadir/validator-deposit>$BASE_PATH/kintsugi/datadir/validator-deposit/logs/deposit_validator_"$RUN_DATE".log 2>&1 & 
+}
+
+function start_eth2stats {
+	echo "starting eth2 stats client..."
+
+	nohup ./bin/eth2stats-client-Linux-x86_64 run \
+	--beacon.type=prysm \
+	--data.folder=$BASE_PATH/kintsugi/datadir/eth2stats \
+	--eth2stats.node-name=atif-l15-test \
+	--eth2stats.addr=34.147.116.58:9090 \
+	--eth2stats.tls=false \
+	--beacon.metrics-addr=http://127.0.0.1:8080/metrics \
+	--beacon.addr=127.0.0.1:4000>$BASE_PATH/kintsugi/datadir/eth2stats/logs/eth2stats_"$RUN_DATE".log 2>&1 & 
+}
+
+function generate_genesis {
+	echo "generating new genesis ssz for phase0..."
+
+	./bin/eth2-testnet-genesis phase0 \
+	--config $BASE_PATH/kintsugi/config/config.yaml  \
+	--mnemonics $BASE_PATH/kintsugi/config/mnemonics.yaml  \
+	--state-output $BASE_PATH/kintsugi/config/genesis.ssz \
+	--tranches-dir  $BASE_PATH/kintsugi/config/tranches \
+	--eth1-block 0x375751608361d3b263737069b0835b41061bcebcfd10c652e4a8bc30676df9ba
+	# --timestamp $(date +%s)
+}
+
+function reset_node {
+	echo "resetting whole node...."
+
+	echo "shutting down beacon client..."
+    sudo kill -INT $(sudo lsof -t -i:4000) &>/dev/null
+    sleep 1
+    sudo kill -INT $(sudo lsof -t -i:12000) &>/dev/null
+    sleep 1
+    sudo kill -INT $(sudo lsof -t -i:13000) &>/dev/null
+    sleep 1
+
+    echo "shutting down validator client..."
+    sudo kill -INT $(sudo lsof -t -i:7000) &>/dev/null
+	sleep 1
+
+	echo "shutting down deposit validator client..."
+    sudo kill -INT $(sudo lsof -t -i:7002) &>/dev/null
+	sleep 1
+
+	echo "shutting down catalyst client..."
+	sudo kill -INT $(sudo lsof -t -i:30303) &>/dev/null
+	sleep 1
+
+	echo "shutting down eth2 stats client..."
+	sudo pkill "eth2stats-client*"
+    sleep 1
+
+    echo "cleanning up datadir..."
+    rm -rf $BASE_PATH/kintsugi/datadir/geth/geth;
+    rm -rf $BASE_PATH/kintsugi/datadir/beacon/beaconchaindata $BASE_PATH/kintsugi/datadir/beacon/metaData;
```

Comment:
You should also delete `tosaccepted` file here: `$BASE_PATH/kintsugi/datadir/beacon/tosaccepted`